nmrfx-processor/src/main/java/org/nmrfx/processor/processing/Processor.java

```diff
@@ -1505,12 +1513,12 @@ public boolean doneWriting() {
      * @param p
      */
     public void run(ProcessOps p) {
+        System.out.println("run " + p.getDim());
```

Comment:
change to log message
```suggestion
        log.info("run {}", p.getDim());
```